components/zcash_protocol/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this library adheres to Rust's notion of
 ### Added
 - `zcash_protocol::PoolType::{TRANSPARENT, SAPLING, ORCHARD}`
 
+### Changed
+- MSRV is now 1.66.0.
```

Comment:
This component crate is an exception, as it was recently extracted from `zcash_primitives` and thus shares a previous MSRV (and is less of a component crate and more of a crate we will regularly be updating, so a synced MSRV makes sense).